docs/debugging.md

```diff
@@ -56,3 +56,9 @@ You *can't*:
 * Perform most other ordinary debugging scenarios.
 
 Development of further debugging scenarios is an on-going focus of the engineering team.
+
+## Troubleshooting tips
```

Comment:
Using "tips" here, but only one tip is provided at this time. Let's go with just "Troubleshooting tip" for the time being. We can pluralize it when a 2nd tip is added (or *more!*).